charter.md

```diff
@@ -1,56 +1,113 @@
-   (such as IRC services, bots, bouncers, libraries);
- * Developers from IRC networks greater than 1,000 active at-peak users which have adopted
-   the IRCv3 specifications.
-
-Other users may participate in the group in a semi-moderated fashion.  We encourage developers of new
-software relating to IRC to participate in the public resources.
-
-## Resources
-
- * The [ircv3/ircv3-specifications](https://github.com/ircv3/ircv3-specifications) Github repository and issue tracker.  The Github repository is the
-   official repository containing the latest version of the official specifications.  Changes are to
-   be submitted using a pull request.  We request that pull requests not be submitted to social
-   media aggregators, as we have no mechanism to moderate the bug tracker efficiently.
-
- * The [#ircv3](irc://irc.freenode.net/ircv3) channel at [irc.freenode.net](irc://irc.freenode.net/ircv3).  This is the main public space for the working group.
-   You are required to follow specific rules to participate in this IRC channel, else you may be
-   removed at any time.
-
-## Ground Rules
-
-To ensure smooth operations of the working group, we request that:
-
- * Participants assume good faith from other participants.  The stakeholders have full moderation
-   privilege over the working group resources and may make moderation decisions if one or more
-   participants is unable to assume good faith.
-
- * Participants do not use social media aggregators to "bomb" working group resources because they
-   do not personally agree with a change being made.
-
- * Participants maintain a respectful and professional atmosphere.  This means no use of coercive
-   threats to force cooperation.
-
- * Participants refrain from soapboxing on the IRC channels.  If there is an issue that needs that
-   level of discussion, a bug or pull request should be filed and discussed on the tracker.
+These extensions are published on this website through backwards-compatible [specifications](/irc/).
+
+
+## Technical board
+
+The direction of these specifications is led by the technical board. The board consists of a broad range of IRC software and specification authors that reflect the community at large. The board will seek to make all decisions by consensus.
+
+Nominations for new members of the board will be considered according to the following criteria
+
+ * Demonstrated support and adoption of the IRCv3 specifications
+ * Developers of widely used IRC software
+ * Administrators of IRC networks with at least 1,000 active peak users
+ * People who have otherwise contributed materially to the specifications
+ * Approval from the existing board
+ * Adherence to the [code of conduct](/conduct.html)
+
+We're seeking representation from as many members of the IRC community as possible. If you or your project would be a good candidate for representation, please contact the board. The current members of the board are:
+
+<table>
```

Comment:
Honestly, when this is merged I'll go through and convert it to autogenerate the tables from some new `_data` lists instead.